### PR TITLE
updated member bios and members

### PR DESCRIPTION
**Pull request by eric-zhou2001**
Contains the updated member bios and list of members.

### CODE REVIEW

File: _data/member_bios.yml [hunk 1]
```diff
@@ -132,7 +132,7 @@ Matthew Kramer:
   connect_uri: 'https://www.linkedin.com/in/matthewthekramer/'
   name: Matthew Kramer
   photo_uri: '/assets/images/members/Kramer, Matthew.jpg'
-  status: President
+  status: Publicity Chair
 Matthew Nicol:
   connect_uri: 'https://www.linkedin.com/in/matthew-nicol-1b9655a4/'
   name: Matthew Nicol
```
Comment (matthewthekramer) [inline _data/member_bios.yml]: I would delete Matt, he graduated a while ago (2017 I believe)

**Comment by matthewthekramer** [comment]
I would also do another round of hassling members for headshots or stealing them from their LinkedIn

**Comment by JoeDailey** [comment]
Any updates here? Should we abandon this or rebase it?

**Review by eric-zhou2001 — COMMENTED** [review]
> Any updates here? Should we abandon this or rebase it?

We can trash it, this was back in the spring and I've since updated the repo! Thanks again for the hard work! :)

**Pull request closed without merging**